two-factor.php

```diff
@@ -15,10 +15,15 @@
  */
 define( 'TWO_FACTOR_DIR', plugin_dir_path( __FILE__ ) );
 
+/**
+ * Version of the plugin.
+ */
+define( 'TWO_FACTOR_VERSION', '0.5.1' );
```

Comment:
Introducing this will add another item to the pre-release checklist to be updated but that's fine since it's in the same file as the main plugin version.